pkg/handlers/generic/mutation/handlers.go

```diff
@@ -30,5 +31,10 @@ func MetaMutators(mgr manager.Manager) []mutation.MetaMutator {
 		mirrors.NewPatch(mgr.GetClient()),
 		calico.NewPatch(),
 		users.NewPatch(),
+
+		// Some patches may have changed containerd configuration.
+		// We must restart containerd for the configuration to take effect.
+		// Therefore, we must apply this patch last.
+		containerdrestart.NewPatch(),
```

Comment:
IIUC the restart will be added all the time, not just when some other patch touches Containerd? Thats totally fine if thats the case, but I think we should leave a comment here about it.

Reply:
yes. We did decided to mimic konvoy behavior where we apply restart regardless any patch touched containerd or not.

Reply:
@dkoshkin WDYT
```go
// Containerd restart and readiness altogether could take ~5s.
// We want to keep patch independent of each other and not share any state.
// Therefore, We must always apply this patch regardless any other patch modified containerd configuration.
```